izanami-frontend/src/pages/webhooks.tsx

```diff
@@ -64,10 +66,17 @@ export function WebHooks(props: { tenant: string }) {
     }
   );
 
+  const [isModalOpen, setModalOpen] = React.useState(false);
+  const [selectedWebhook, setSelectedWebhook] = React.useState<Webhook>();
```

Comment:
Rather than duplicating this kind of behaviour in several components, it would be nice to develop an askPasswordConfirmation function in context, that takes a message, prompt user for password, and runa. given promise on confirmation, closing only if request succeeds